packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -154,283 +151,230 @@ function checkForGraphFragmentPattern(
   considerWriteOnly,
   schemaFinder
 ) {
-  let variantOmitsProperties = false;
+  // Invoke primary algorithm.
+  const variantIsGraphFragment = isGraphFragment(
+    variant,
+    canonical,
+    canonicalPath,
+    false,
+    considerWriteOnly,
+    schemaFinder
+  );
 
-  // The function is wrapped in a partial so that we can track `variantOmitsProperties`
```

Comment:
Apologies for the large diff in this PR. _Most_ of the changes are just moving this function, as it no longer needs to be a partial.